.github/workflows/main.yml

```diff
@@ -65,16 +65,25 @@ jobs:
       test: style
 
   test:
-    name: Gtests (Sanity)
+    name: Gtests
     needs: build
     uses: ./.github/workflows/test.yml
     with:
       name: 'gcc-cpu'
       cache-suffix: ${{ needs.build.outputs.cache-suffix }}
       test: sanity
 
+  test-modin:
+    name: Python tests
+    needs: build
+    uses: ./.github/workflows/test.yml
+    with:
+      name: 'gcc-cpu'
+      cache-suffix: ${{ needs.build.outputs.cache-suffix }}
+      test: modin
```

Comment:
With single job we actually mixing pyhdk api test and modin check. @ienkovich  PTAL. 
I think it's fine, but there may be a need to see them separately.